src/features/landing/components/Possibilities/CardList/CardList.tsx

```diff
@@ -20,7 +20,12 @@ function CardList() {
         ))}
       </Grid>
       <Grid sx={cardListStyles.gridMobile}>
-        <Swiper pagination modules={[Pagination]} className="swiper-wrapper">
+        <Swiper
+          pagination
+          modules={[Pagination]}
+          spaceBetween={20}
+          className="swiper-wrapper"
+        >
           {UNLIMITED_INTEGRATIONS_CARD_ITEMS.map(item => (
```

Comment:
on the previous call, we discussed this constant, I don't see that it was removed from this component, read about SOLID and remove this constant from this component